praetorian_cli/ui/aegis/commands/job.py

```
@@ -226,6 +243,226 @@ def run_job(menu, args):
+
+        for prefix in search_prefixes:
+            try:
+                files, _ = menu.sdk.files.list(prefix_filter=prefix)
+                if files:
+                    for f in files:
+                        fkey = f.get('key', '')
+                        if fkey not in seen_keys:
+                            seen_keys.add(fkey)
+                            all_files.append(f)
+            except Exception:
+                pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)